app/controllers/concerns/client_sortable.rb

```diff
@@ -24,38 +24,52 @@ def filtered_and_sorted_clients(default_order: nil)
 
   # see if there are any overlapping keys in the provided params and search/sort set
   def has_search_and_sort_params?
-    (params.keys.map(&:to_sym) & search_and_sort_params).any?
+    overlapping_keys = (params.keys.map(&:to_sym) & search_and_sort_params)
```

Comment:
100% not a blocker: I hope when we upgrade to Ruby 2.7, we can switch this to `.intersection` so I can forget what `&` means in this context. :)